src/components/LocationPage/NewLocationPageHeader.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { Fragment } from 'react';
+import CheckIcon from '@material-ui/icons/Check';
+import SummaryStats from 'components/SummaryStats/SummaryStats';
+import {
+  ColoredHeaderBanner,
+  HeaderContainer,
+  HeaderTitle,
+  HeaderSection,
+  HeaderSubCopy,
+  HeaderSubtitle,
+  ButtonsWrapper,
+  HeaderButton,
+  LocationCopyWrapper,
+  HeaderSubCopyWrapper,
+  LastUpdatedDate,
+  RiskLevelThermometer,
+  RiskLevelWrapper,
+  RiskLevelTitle,
+  RiskLevel,
+  RiskLevelGraphicMobile,
+  RiskLevelGraphicDesktop,
+  Triangle,
+} from 'components/LocationPage/NewLocationPageHeader.style';
+import { useEmbed } from 'common/utils/hooks';
+import { LOCATION_SUMMARY_LEVELS } from 'common/metrics/location_summary';
+import { Level } from 'common/level';
+import { COLOR_MAP } from 'common/colors';
+import { useModelLastUpdatedDate } from 'common/utils/model';
+import { STATES_WITH_DATA_OVERRIDES } from 'common/metrics/hospitalizations';
+import { Projections } from 'common/models/Projections';
+import { formatDate } from 'common/utils';
+import { isNull } from 'util';
+
+const RiskLevelGraphic = (props: {
+  projections: Projections;
+  alarmLevel: number;
+  levelName: string;
+}) => {
+  return (
+    <RiskLevelWrapper>
+      <RiskLevel>{props.levelName}</RiskLevel>
+      <RiskLevelThermometer alarmLevel={props.alarmLevel} />
+      <Triangle alarmLevel={props.alarmLevel} />
+      <RiskLevelTitle>Covid Risk Level</RiskLevelTitle>
+    </RiskLevelWrapper>
+  );
+};
+
+const LocationPageHeading = (props: { projections: Projections }) => {
+  const { isEmbed } = useEmbed();
+
+  const displayName = props.projections.countyName ? (
+    <>
+      <strong>{props.projections.countyName}, </strong>
+      <a
+        href={`${
+          isEmbed ? '/embed' : ''
+        }/us/${props.projections.stateCode.toLowerCase()}`}
+      >
+        {props.projections.stateCode}
+      </a>
+    </>
+  ) : (
+    <strong>{props.projections.stateName}</strong>
+  );
+
+  return <span>{displayName}</span>;
+};
+
+const noop = () => {};
+
+const NewLocationPageHeader = (props: {
```

Comment:
What's the rationale for having NewLocationPageHeader.tsx and still keeping LocationPageHeader.js ?  Just temporary?  Should we have a TODO to delete LocationPageHeader.js and rename NewLocationPageHeader?

Reply:
Yep! Just temporary. I plan on removing the old header+old embed after everything is deployed smoothly. I added in a TODO